core/src/main/java/com/bankbazaar/tripmanager/controller/RegisterController.java

```diff
@@ -0,0 +1,53 @@
+package com.bankbazaar.tripmanager.controller;
+
+
+import com.bankbazaar.tripmanager.manager.RegisterManager;
+import com.bankbazaar.tripmanager.model.Users;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+
+import static org.hibernate.tool.schema.SchemaToolingLogging.LOGGER;
+
+@Controller
+@RequestMapping("/register")
+public class RegisterController {
+    @Autowired()
+    private RegisterManager service;
+
+    @Autowired
+    protected AuthenticationManager authenticationManager;
+
+    @RequestMapping(method = RequestMethod.GET)
+    public String welcome(){
+        return "register";
+    }
+    @RequestMapping(method = RequestMethod.POST)
+    public String addUsers(@Valid @ModelAttribute Users user, HttpServletRequest request) {
+        String plainPassword = user.getPassword();
+        Users response = service.saveUsers(user);
+        if(response==null)
+        {
+            return "email_already_registered";
```

Comment:
This should be shown as an error message in the screen

Reply:
I have implemented in such a way that the alert message is shown in the same page.